src/App.js

```diff
@@ -563,9 +564,9 @@ class Loader extends React.Component<Props, State> {
 
   onSelectWheelchairAccessibility = (value: YesNoLimitedUnknown) => {
     if (this.props.featureId) {
-      this.props.history.push({
-        pathname: `/beta/nodes/${this.props.featureId}/edit-wheelchair-accessibility`,
-        search: `presetStatus=${value}`,
+      this.setState({
+        modalNodeState: 'edit-wheelchair-accessibility',
```

Comment:
-1: Should the `modalNodeState` be part of the actual route so we do not have two state machines but one based on routing?

Reply:
Last discussion status with @opyh & you on Thursday was that we wanted to remove this from the routes. I guess we can bring all of that back, but its unrelated to the fix, the removed lines don't even work.

Reply:
Really? Than I changed my mind. Anyway, you're right. This is probably not part of this PR.

Reply:
I am all in for fewer state machines!

Reply:
Should we add a ticket in Trello?